BLSQ/iaso
# IA-1742: Page edition users dropdown error

Previous code was removing current user from users dropdown options while displaying list of allowed users that can access a page. 
<img width="1114" alt="Screenshot 2023-01-09 at 12 49 36" src="https://user-images.githubusercontent.com/12494624/211301635-5d362068-ce6a-44e5-803e-53219d6c852c.png">

## Self proof reading checklist


## Changes

Refactor users selection using renderTags and disable user deletion on current user.

## How to test

Edit a page.

Check the Ui in not displaying an error, try to delete yourself in the users dropdown

## Print screen / video


https://user-images.githubusercontent.com/12494624/211302038-3afb23a9-8d24-4d12-af3b-7581f47165c2.mov
